src/front-end/src/components/MentorForm.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import { Form, Button, Input, DatePicker, Tooltip, Select } from 'antd';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { InfoCircleOutlined, QuestionCircleOutlined } from '@ant-design/icons';
+import TimezonePicker from 'react-bootstrap-timezone-picker';
+import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
+import TextArea from 'antd/lib/input/TextArea';
+
+const { RangePicker } = DatePicker;
+
+const { Option } = Select;
+
+class MentorForm extends Component {
+  render() {
+    return (
+    <div className="MentorForm"> 
+      <Form>
+      <h1>
+        Mentor is .......

```

Comment:
I don't understand this text. I think we should replace it with "Sign up as a mentor".

Reply:
done